### PR TITLE
Fix ova artifact link structure

### PR DESCRIPTION
**Pull request by mitalipaygude**
*Issue #, if available:*

*Description of changes:*
Fixing the artifact link structure for ovas. 

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/pkg/assets_eksd.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (r *ReleaseConfig) GetEksDChannelAssets(eksDReleaseChannel, kubeVer, eksDRe
 
 			if r.DevRelease || r.ReleaseEnvironment == "development" {
 				sourceS3Key = fmt.Sprintf("%s.%s", osName, imageExtensions[imageFormat])
-				sourceS3Prefix = fmt.Sprintf("%s/%s/%s/%s/%s", imageBuilderProjectPath, eksDReleaseChannel, imageFormat, osName, latestPath)
+				sourceS3Prefix = fmt.Sprintf("%s/%s/%s/%s/%s", imageBuilderProjectPath, imageFormat, eksDReleaseChannel, osName, latestPath)
```
Comment (danbudris) [inline release/pkg/assets_eksd.go:71]: did we look to see if the dev release structure will break anything?

Comment (mitalipaygude) [reply release/pkg/assets_eksd.go:71]: https://tiny.amazon.com/v5ha8lwx/IsenLink

Comment (mitalipaygude) [reply release/pkg/assets_eksd.go:71]: it was wrong as well.

Comment (vivek-koppuru) [reply release/pkg/assets_eksd.go:71]: Was taking a look but can't find the artifact listed under this: https://github.com/aws/eks-anywhere/blob/5fc329d3390156532eebc0ebfd29aa65cf8f4a63/release/pkg/assets_eksd.go#L71

Comment (vivek-koppuru) [reply release/pkg/assets_eksd.go:71]: Let's not change this just because the imageBuilderProjectPath prefix isn't something we are able to find right now

**Review by vivek-koppuru — APPROVED** [review]
we can figure out the other paths later

**Pull request merged**